pkg/gcp/gcp.go

```diff
@@ -25,7 +25,6 @@ import (
 const (
 	GCloudExecutable = "gcloud"
 	gSUtilExecutable = "gsutil"
```

Comment:
Unrelated to the PR, but can someone fix the capitalization on this var?
Should be `gsutilExecutable`.

Reply:
Yes, I can follow-up on this one. :+1: